### DISCUSSION TITLE
Weird behaviour in simulation run

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1668] [Q&A] [answered]
Hi, I'm trying to run a simulation which is pretty simple and that worked quiet fine until i made few modification, changing some elements. I notice that the time of the simulation decrease for no reason (no error recorded) and as a result, the estimate time of the simulation doesn't make any sense. Any suggestions?
[crash_test_A01.txt](https://github.com/OpenRadioss/OpenRadioss/files/11948176/crash_test_A01.txt)

**Comment by MarianBulla** [comment]
Hi,
thank you for communicating this. 
And, yes this is possible. 
The calculation time depends on many things.
The estimate time is just an estimation. It can change significantly, if something happens within the model, during the simulation.
It can not be forseen, infront.

Anyway, sudden changes in the timestep (and therefore on the calculation time) are indicating, that there is something significantly happening during your simulation.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by PaulAltair** [reply]
I ran your model and it proceeded as expected, are you simply referring to the estimated time remaining value? This will be initially quite inaccurate: it is doing nothing more sophisticated than extrapolating a total time from the time take to complete cycles to that point, so the first few guesses will be 'bad', after some cycles you should see it converge to a sensible value

**Reply by FGarelli** [reply]
I wasn't referring to the time estimate. It looks that the time reach during the simulation go further and then backward at the beginning. I could complete the simulation but I wonder why I notice this behaviour.

**Reply by PaulAltair** [reply]
I'm sorry, I don't understand what you mean then?, can you share a screenshot indicating where you think there is the issue? After how long are you seeing the unexpected behaviour? I have started run again here and it looks ok to me: (except.. my laptop is a little overloaded! hence the long runtime estimate!)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/91537004-4046-4bcd-8ca3-63ed7a4c83df)

**Reply by FGarelli** [reply, marked as answer]
The simulation is running, i just notice that the time that i highlight is going backward at some point and then again forward. I was just wandering why. 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/15f7561f-8a9e-46b2-b8bc-5aefd455d337)

**Reply by elequiniou** [reply]
Hi,

I hope you are not blocked waiting further answer, the simulation time is not going backward, but seems to progress smoothly.

Thanks,

Eric
